docs/chains/getting-started.mdx

```diff
@@ -14,50 +14,57 @@ description: "Getting Started with *Truss Chains*"
 * [Reference](/chains/full-reference)
 
 
-This guide shows you how to install Truss chains, create your first chain and run it.
+This guide shows you how to install Truss Chains, create your first Chain and run it.
 
 # Installation
 
-Truss chains is available in the a rc version of Truss.
+Truss Chains is available in the a rc version of Truss.
 
 ```bash
-$ pip install truss==0.9.10rc3
+$ pip install truss==0.9.11rc1
 ```
 
-## Baseten Setup
+For deploying Chains remotely to Baseten you need a [Baseten account](https://app.baseten.co/signup).
 
-* Make sure you have an active baseten-truss login in your environment, i.e. a valid API key in your `.trussrc`.
-* Add `baseten_chain_api_key` to [Baseten Secrets](https://app.baseten.co/settings/secrets) setting the token value
- to a baseten API key that has permissions to deploy models (e.g. the same key as in your `.trussrc`).
 
+# Creating your first *Chainlet*
```

Comment:
we can do this separately, but I think it would be great if the first example had two chainlets